app/src/main/java/nu/yona/app/api/manager/AuthenticateManager.java

```diff
@@ -93,6 +96,8 @@
 	 */
 	void getUser(final String url, final DataLoadListener listener);
 
+	JSONObject getStoredUserObjectFromDB();
```

Comment:
This one we don't need, see my comment on ``getStoredUserObjectFromDB``

Reply:
Done. removed.